hydra_provider/Cargo.toml

```diff
@@ -6,14 +7,19 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-messages = { git = "https://github.com/uorocketry/hydra", branch = "master" }
+messages = { git = "https://github.com/uorocketry/hydra", branch = "messages-improvements", features = ["ts"] }
+mavlink = { git = "https://github.com/uorocketry/rust-mavlink", branch = "hydra_dialect", features = ["default"] }
```

Comment:
Should we merge hydra_dialect to master of the rust-mavlink fork?

Reply:
not yet, once we are sure of our dialect